unblob/report.py

```diff
@@ -236,6 +236,29 @@ class UnknownChunkReport(Report):
     entropy: Optional[EntropyReport]
 
 
+@attr.define(kw_only=True, frozen=True)
+class CarveReport(Report):
+    """Describes a successful carving operation."""
+
+    carved_from: Path
+    carved_to: Path
+    start_offset: int
+    end_offset: int
+    handler_name: str
+
+    @classmethod
+    def from_chunk(
+        cls, carved_from: Path, carved_to: Path, chunk: "ValidChunk"  # noqa: F821
+    ):
+        return cls(
+            carved_from=carved_from,
+            carved_to=carved_to,
+            start_offset=chunk.start_offset,
+            end_offset=chunk.end_offset,
+            handler_name=chunk.handler.NAME,
```

Comment:
I had another look at this implementation and its report-output, and I do not see how duplicating the chunk attributes is useful.

This is also storing the path to the carved file, while what was missing is some record for the parent directory, which is created to hold the carved files and their extractions.

Maybe I miss some use-case for the stored attributes, but I am inclined to drop this `CarveReport` and introduce instead a `CarveDirectoryReport`, with the missing carve directory path.